test/image/mocks/gl3d_tetrahedra.json

```diff
@@ -16,6 +16,8 @@
         [1, "rgb(0, 0, 255)"]
       ],
       "intensity": [0, 0.33, 0.66, 1],
+      "cmin": -0.5,
+      "cmax": 0.5,
```

Comment:
Weird. That made the axis labels disappear. I might have to revert back those `gl-error3d` dependencies bump of commit https://github.com/gl-vis/gl-error3d/commit/d756284bbfdaf3dd1f4ce7e69ae8085b9b6d2a64